src/ui/nypl-base-theme/foundations/breakpoints.ts

```diff
@@ -1,10 +1,10 @@
 // theme.js
-import { createBreakpoints } from '@chakra-ui/theme-tools';
-
-export default createBreakpoints({
```

Comment:
Just wondering, why remove the `createBreakpoints` function? It's only really there to aid in type checking

Reply:
There was a warning that this is being deprecated by Chakra soon and to just pass the breakpoints object to the theme.

Reply:
Sweet thanks for the explanation!